pkgs/by-name/py/pytr/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "pytr";
+  version = "2024-04-01-843ba36";
```

```suggestion
  version = "0.1.9-unstable-2024-04-01";
```

per https://github.com/NixOS/nixpkgs/tree/master/pkgs:
> - The version attribute must start with a digit e.g., "0.3.1rc2".
>
> - If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.
>
> Example: Given a project had its latest releases 2.2 in November 2021, and 3.0 in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release 2.2.1 would have version = "2.2-unstable-2022-03-15".
>
>- If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be 0.

---

Thank you, wasn't able to find this.